packages/lib/CalEventParser.ts

```diff
@@ -176,7 +176,11 @@ export const getRescheduleLink = (calEvent: CalendarEvent): string => {
   return `${calEvent.bookerUrl ?? WEBAPP_URL}/reschedule/${seatUid ? seatUid : Uid}`;
 };
 
-export const getRichDescription = (calEvent: CalendarEvent, t_?: TFunction /*, attendee?: Person*/) => {
+export const getRichDescription = (
```

Comment:
Can we make it accept an object with these props to improve readability at the place where we are calling it.